Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -189,46 +192,25 @@ const SubmissionDetails = () => {
         [ currentSubmission, getSubmissionDetailsResults ],
     );
 
-    const renderReloadButton = useCallback(
-        () => (
+    const renderButtonsSection = useCallback(() => (
+        <div className={styles.buttonsSection}>
             <Button
               onClick={handleReloadClick}
               text="Reload"
               type={ButtonType.secondary}
-              className={submissionsReloadBtnClassName}
+              className={styles.submissionReloadBtn}
             />
-        ),
-        [ handleReloadClick ],
-    );
-
-    const renderRetestButton = useCallback(
-        () => {
-            if (!canAccessAdministration) {
-                return null;
-            }
-
-            return (
+            { canAccessAdministration && (
```

Comment:
This is odd. Boolean && component?

Reply:
why would it be odd? it's a conditional render statement, it would only render the component if the statement returns truthy value

Reply:
Isnt it more easily readable to write it this way (pseudo code):
```
ifTruth
? renderComponent
: null
```

Reply:
a matter of preference, if you want I can change it like that, it basically has no difference, for me its easier to read with && if we have no else statement, but it literally makes no difference, so whatever you say :D